Shopstagram - rozwiązanie/css/base/_colors.scss

```diff
@@ -0,0 +1,11 @@
+$color-white: #ffffff;
+$color-black: #000000;
+$color-primary: #fafafa;
+$color-secondary: #efefef;
+$color-user-img: #DD2A7C;
```

Comment:
nazwa koloru powinna byc bardziej uniwersalna

okreslajac nazwe koloru jako "color-img" albo "color-border" praktycznie zamykasz sobie mozliwosc uzycia go gdzie kolwiek indziej

podsylam artykul / sciage: 
https://css-tricks.com/what-do-you-name-color-variables/

warto zwrocic uwage na PDF z kolorami jaki wam wyslalem do shopstagrama, zobaczysz tam ze 3 albo 4 z nich sa roznymi odcieniami szarosci, wiec mozna je np nazwac

color-neutral
color-neutral--light
color-neutral--dark

albo nawet w ostatecznosci

color-gray
color-gray--light
color-gray--dark